crates/bevy_window/src/window.rs

```diff
@@ -535,6 +535,13 @@ pub struct WindowDescriptor {
     pub cursor_visible: bool,
     pub cursor_locked: bool,
     pub mode: WindowMode,
+    /// Sets whether the background of the window should be transparent.
+    /// # Platform-specific
+    /// - iOS / Android / Web: Unsupported.
+    /// - OSX / Linux : Not working as expected.
+    /// OSX transparent works with winit out of the box, so this issue might be related to: https://github.com/gfx-rs/wgpu/issues/687
+    /// Linux now works with this pr merged in but bevy is using an older version of winit: https://github.com/rust-windowing/winit/pull/2006
```

Comment:
Bevy is using the latest released version of Winit, there hasn't been a release since this PR was merged

Reply:
Sorry, I should of said master branch of winit, my mistake. I'll update the comment now